### PR TITLE
better usage of typescript

### PR DESCRIPTION
**Pull request by mod7ex**
I looked at this example and typescript there isn't correctly used i think i made it a bit better

### CODE REVIEW

File: examples/with-static-export/@types/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface IPost {
```
Comment (styfle) [inline examples/with-static-export/@types/global.d.ts:1]: image

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names

**Review by styfle — CHANGES_REQUESTED** [review]
image

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names

**Comment by modex98** [comment]
i fixed the naming issue, thanks

File: examples/app-dir-i18n-routing/app/[lang]/page.tsx [hunk 2]
```diff
@@ -3,19 +3,15 @@ import { Locale } from '../../i18n-config'
...
-}) {
+export default async function IndexPage({ params: { lang } }: { params: { lang: Locale } }) {
   const dictionary = await getDictionary(lang)
 
   return (
     <div>
       <LocaleSwitcher />
       <p>Current locale: {lang}</p>
       <p>
-        This text is rendered on the server:{' '}
```
Comment (styfle) [inline examples/app-dir-i18n-routing/app/[lang]/page.tsx:18]: Why was the space removed?

Comment (modex98) [reply examples/app-dir-i18n-routing/app/[lang]/page.tsx:18]: ah yes i'm sorry i just used VS code for the first time to contribute and ...))

**Review by puneet2407 — APPROVED** [review]

**Comment by balazsorban44** [comment]
Thanks for the PR, but this example has been converted to the App Router, so this pr should not be merged anymore. See https://github.com/vercel/next.js/pull/47584

**Pull request closed without merging**